cmd/kaniko-acr/main.go

```diff
@@ -308,15 +309,27 @@ func getACRToken(tenantId, clientId, clientSecret, cert, registry string) (strin
 	}
 
 	// TODO check for presence of file as well.
-	os.Setenv(clientIdEnv, clientId)
-	os.Setenv(clientSecretKeyEnv, clientSecret)
-	os.Setenv(tenantKeyEnv, tenantId)
+	err := os.Setenv(clientIdEnv, clientId)
```

Comment:
if err := os.Setenv(clientIdEnv, clientId); err != nil {
  return errors.Wrap(err, "failed to set env variable client Id")
}

Reply:
Same for all other cases

Reply:
done